src/Agent.Sdk/SecretMasking/OssSecretMasker.cs

```diff
@@ -13,6 +15,28 @@ namespace Agent.Sdk.SecretMasking;
 public sealed class OssSecretMasker : ISecretMasker, IDisposable
 {
     private SecretMasker _secretMasker;
+    private Telemetry _telemetry;
+
+    /// <summary>
+    /// The maximum number properties to report in a single
+    /// 'SecretMaskerDetections' telemetry event. This value was chosen to keep
+    /// the size in the range of existing routine telemetry events.
+    /// </summary>
+    public const int MaxDetectionsPerTelemetryEvent = 20;
+
+    /// <summary>
+    /// The maximum number of 'SecreMaskerDetections' telemetry events to send.
+    /// If this would be have to be exceeded to record all unique C3Id, the
+    /// overall 'SecretMasker' event will indicate this by including a
+    /// 'DetectionDataIsIncomplete' property set to 'true'.
```

Comment:
>property [](http://example.com/codeflow?start=36&length=8)

maybe clarify by property here you are referring to a telemetry not a code property.

Reply:
This comment is gone now, but probably should come back somewhere and if so, remember to take this note.